job_complete_tx.go

```diff
@@ -50,6 +50,9 @@ func JobCompleteTx[TDriver riverdriver.Driver[TTx], TTx any, TArgs JobArgs](ctx
 	if err != nil {
 		return nil, err
 	}
+	if len(rows) == 0 {
+		return nil, rivertype.ErrNotFound
+	}
```

Comment:
Uncovered this bug while thinking about how `JobCompleteTx` would function if called from the new test helpers. Will add it to the changelog.